### PR TITLE
Add visual tests for the feedback page footer

### PR DESCRIPTION
**Pull request by pablothedude**
In order to detect regressions in the footer of the feedback page
some tests are added.

### CODE REVIEW

File: theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        for (const footer of footerDifferences) {
+            for (const viewport of viewports) {
+                it(`${footer.name}-${viewport.width}x${viewport.height}`, async () => {
+
+                    await page.goto(footer.url);
+                    await page.setViewport(viewport);
+                    await page.waitFor(".error-container");
+                    const screenshot = await page.screenshot({
+                        fullPage: true,
+                        path: `./material/javascripts/tests/visual-regression/error-page/screenshots/footer/${footer.name}-${viewport.width}x${viewport.height}.png`
```
Comment (MKodde) [inline theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js]: can you add the `footer` folder by adding a .gitkeep file in that dir?

File: src/OpenConext/EngineBlockFunctionalTestingBundle/Controllers/FeedbackController.php [hunk 2]
```diff
@@ -57,8 +57,6 @@ public function feedbackAction(Request $request)
         $feedbackInfo = $this->getFeedbackInfo($request);
         $parameters = $this->getTemplateParameters($request);
 
-        $this->validateFooterTemplateParameters($parameters);
```
Comment (MKodde) [inline src/OpenConext/EngineBlockFunctionalTestingBundle/Controllers/FeedbackController.php]: Why did you remove the validate method?

File: theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js (hunk 1 above)
Comment (pablothedude) [reply theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js]: Discussed IRL and decided to remove the .gitignore from the theme folder and used the one in the root of the project.

File: src/OpenConext/EngineBlockFunctionalTestingBundle/Controllers/FeedbackController.php (hunk 2 above)
Comment (pablothedude) [reply src/OpenConext/EngineBlockFunctionalTestingBundle/Controllers/FeedbackController.php]: Because it wasn't needed after all because of the `is defined` logic in the footer template. So the parameters are not required.

File: theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js (hunk 1 above)
Comment (MKodde) [reply theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js]: Moving the gitignore will not fix this problem, I suspect. The test fails if the folder is not present, my proposal is to add the empty folder with a .gitkeep file to prevent end users from having to create directories to get tests to pass.

**Comment by MKodde** [comment]
Like in #691, is the change to the Package.json a requirement? If so, please elaborate.Otherwise please drop it from the PR.

File: theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js (hunk 1 above)
Comment (pablothedude) [reply theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js]: I lacked to push the change, done.

**Comment by pablothedude** [comment]
Reverted the package-lock.json change

File: theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js (hunk 1 above)
Comment (pablothedude) [reply theme/material/javascripts/tests/visual-regression/error-page/Footer.test.js]: I lacked to push the change, done.
